### ISSUE TITLE
Train Dataset Question

### ISSUE THREAD

**Issue by seTalent** [vedernikovphoto/TCCT_Net#1]
Thanks for your good job!

I wonder to know whether there is error in the train_label.xlsx.
When I get the EmotiW Dataset, i found some labels in the  train_dataset are NSP(not in [Engaged, Not-Engaged, High-Engaged, Barely-Engaged]).I tried emailing the dataset owner but didn't get a reply.
Is this true for your dataset as well?How did you deal with it?Could you please give me some suggestion?
Thanks!
